src/components/Signup/ShowNewPod.jsx

```diff
@@ -32,6 +34,10 @@ const ShowNewPod = ({ oidcIssuer }) => {
           ? `You have registered with ${data[0].person}`
           : 'You have not registered with a case manager'}
       </Typography>
+      <Typography>Click here to return to the home page and log into PASS:</Typography>
```

Comment:
Maybe upon successful registration it could countdown and then redirect automatically? I've seen plenty of instances along those lines (e.g. "[action successful], you will be redirected in [X] seconds or click here")

Reply:
I agree we can definitely improve this page. It's a placeholder right now before we settle on a real design. A timed countdown is one approach.

I wrote up an issue here related to it:
https://github.com/codeforpdx/PASS/issues/441

One of the ideas I had was to include an overview of what the pod is, and what kind of things it lets you do. This wouldn't work with the timer idea, since the overview would take a lot of time to read, and users may want to be linked to other resources to read more.

Either way it's not related to this PR. We can continue the conversation on that issue.